tests/streams/stringifier-stream.js

```diff
@@ -58,15 +62,23 @@ describe('Stringifier Stream', () => {
                 data += chunk;
             });
             stream.on('end', () => {
-                assert.equal(data, '<html><head></head><body>12</body></html>');
+                assert.equal(
+                    data,
+                    '<html><head></head><body>123</body></html>'
+                );
                 done();
             });
             nodes.forEach(node => stream.write(node));
             stream.end();
             setTimeout(() => {
                 streams[0].end('1');
             }, 10);
-            streams[1].end('2');
+
```

Comment:
I would make this test more complex with more pieces of data on each stream.

Reply:
Will add more tests.

Reply:
Writing some logic to add random delays to the stream.